tools/manifest-templator/manifest-templator.go

```diff
@@ -120,6 +121,7 @@ func main() {
 	runbookURLTemplate := flag.String("runbook-url-template", "", "")
 	featureGates := flag.String("feature-gates", "", "")
 	infraReplicas := flag.Uint("infra-replicas", 0, "")
+	testImageReplicas := flag.String("test-image-replicas", "0", "")
```

Comment:
nit: Should this default to `6`?

Reply:
I don't think so, 6 is like a CI default only for kubevirt/kubevirt